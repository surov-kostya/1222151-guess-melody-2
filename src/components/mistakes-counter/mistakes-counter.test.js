import React from 'react';
import MistakesCounter from './mistakes-counter';
import renderer from 'react-test-renderer';

it(`MistakesCouner renders correctly`, () => {
  const tree = renderer.create(<MistakesCounter mistakes={2}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
