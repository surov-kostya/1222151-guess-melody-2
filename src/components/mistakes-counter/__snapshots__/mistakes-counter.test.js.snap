// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MistakesCouner renders correctly 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
